about.md

```diff
@@ -4,18 +4,53 @@ title: About
 is_nav: true
 permalink: /about/
 ---
+開発、改善、海賊
```

Comment:
Starting with a language most people don't understand is a terrible idea. Sorry.

It __might__ add an air of mystery, but it's not worth the cons of confusing, distracting and generally disconnecting people from the message we actually want to get across.